target/linux/ramips/mt7621/base-files/lib/upgrade/platform.sh

```diff
@@ -51,6 +51,9 @@ platform_do_upgrade() {
 	case "$board" in
 	ampedwireless,ally-00x19k|\
 	ampedwireless,ally-r1900k|\
+	arcadyan,wg630223)
+		nand_do_upgrade "$1"
+		;;
```

Comment:
This will brake other devices. You need only one line here.
`arcadyan,wg630223|\`